instrumentation/net/http/otelhttp/test/handler_test.go

```diff
@@ -141,3 +142,47 @@ func TestHandlerRequestWithTraceContext(t *testing.T) {
 	assert.NotEmpty(t, spans[0].Parent().SpanID())
 	assert.Equal(t, spans[1].SpanContext().SpanID(), spans[0].Parent().SpanID())
 }
+
+func TestWithPublicEndpoint(t *testing.T) {
+	spanRecorder := tracetest.NewSpanRecorder()
+	provider := sdktrace.NewTracerProvider(
+		sdktrace.WithSpanProcessor(spanRecorder),
+	)
+	remoteSpan := trace.SpanContextConfig{
+		TraceID: trace.TraceID{0x01},
+		SpanID:  trace.SpanID{0x01},
```

Comment:
Why not set the `Remote` field here?

```suggestion
		SpanID:  trace.SpanID{0x01},
		Remote: true,
```

Reply:
Missed that, thank you!
Fixed